Assets/PatchKit Patcher/Scripts/AppData/Local/LocalMetaData.cs

```diff
@@ -106,16 +142,17 @@ private void LoadData()
                 DebugLogger.Log("Cannot load from file.");
 
                 LoadEmptyData();
-
             }
         }
 
         private void LoadEmptyData()
         {
-            _data = new Data
-            {
-                FileVersionIds = new Dictionary<string, int>()
-            };
+            _data = JsonConvert.DeserializeObject<Data>("{}"); // Json Deserializer will fill default property values defined in struct
+            _data.FileVersionIds = new Dictionary<string, int>();
+
+#if UNITY_5_3_OR_NEWER // LEGACY: fill productKey from unity prefs
+            _data.productKey = UnityEngine.PlayerPrefs.GetString(ValidateLicenseCommand.CachePatchkitKey, string.Empty);
```

Comment:
1. It's not a good place for loading license key from registry. LoadEmptyData() should fill data with empty values, it should't try and load any valid data even from previous location.
2. PlayerPrefs license key should be removed on successful load.